examples/c99/z_scout.c

```diff
@@ -62,8 +62,8 @@ void fprinthello(FILE *stream, const z_hello_t *hello) {
     fprintf(stream, " }");
 }
 
-void callback(z_owned_hello_t hello, void *context) {
-    fprinthello(stdout, hello._value);
+void callback(z_owned_hello_t *hello, void *context) {
+    fprinthello(stdout, hello->_value);
```

Comment:
same as c11 scouting example

Reply:
In Pico we have it, so I am just using the z_loan now.